indico/web/client/styles/legacy/Default.css

```diff
@@ -788,6 +788,8 @@ body.cke_show_borders {
 .material_tip {
   border-color: #ddd !important;
   box-shadow: 1px 1px 5px rgba(100, 100, 100, 0.2);
+  max-height: 500px;
+  overflow-y: scroll;
```

Comment:
doesn't this mean there are *always* scroll bars? IIRC it's `overflow-y: auto` if you just want them to show up when needed